setup_cfg_fmt.py

```diff
@@ -198,15 +198,21 @@ def _python_requires(
 
 def _requires(cfg: configparser.ConfigParser, which: str) -> List[str]:
     raw = cfg.get('options', which, fallback='')
+    stripped = raw.strip()
+    if not stripped:
+        return []
 
-    install_requires = raw.strip().splitlines()
+    if raw.startswith(os.linesep):
+        install_requires = stripped.splitlines()
+    else:
+        install_requires = [r.strip() for r in stripped.split(';')]
```

Comment:
I wouldn't bother with this, AFAIK this weird semicolon behavior isn't documented anywhere

Reply:
OK